elements/inline-audio/src/inline-audio.js

```diff
@@ -29,7 +31,7 @@ Custom property | Description | Default
  * @element inline-audio
  * 
  */
-class InlineAudio extends I18NMixin(SimpleColors) {
+class InlineAudio extends I18NMixin(SimpleColors, DDD) {
```

Comment:
`I18NMixin(DDD)` and not `SimpleColors,DDD`

Make sure to fix this in the other elements where it's happened